libbluetoe.h

```diff
@@ -0,0 +1,72 @@
+/*
+ Name:		libbluetoe.h
+ Created:	6/18/2018 3:24:17 PM
+ Author:	Tony
+ Editor:	http://www.visualmicro.com
+*/
+
+#ifndef _libbluetoe_h
+#define _libbluetoe_h
+
+#pragma pack(1)
```

Comment:
This one is extremely problematic because it will most likely not be supported by all compilers and much more important: Every file that includes this header will have this pragma applied, which will lead to massive performance degradations on most platforms and larger code size because of the possible, not natural alignment.